client/.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended"
+  ],
```

Comment:
there is prettier plugin for eslint, that can be good

Reply:
Okay, I will add it.